projects/organization-management/src/app/models/node/node.helper.ts

```diff
@@ -0,0 +1,106 @@
+import { NodeData } from './node.interface';
+import { Node, NodeTree } from './node.model';
+
+export class NodeHelper {
+  static rootsFirst(a: NodeData, b: NodeData): number {
+    if (a.relationships.parentNode && !b.relationships.parentNode) {
+      return -1;
+    }
+    if (!a.relationships.parentNode && b.relationships.parentNode) {
+      return 1;
+    }
+    if (!a.relationships.parentNode && !b.relationships.parentNode) {
+      return 0;
+    }
+    if (a.relationships.parentNode && b.relationships.parentNode) {
+      return 0;
+    }
```

Comment:
what's that in binomical relation? `(!identity) && ??` There's room for simplification, right?

Reply:
Once you explored the space and know the tools, there's always room for simplification.